include/sys/fs/zfs.h

```diff
@@ -756,6 +762,15 @@ typedef enum pool_state {
 } pool_state_t;
 
 /*
+ * mmp state. The following states provide additional detail describing
+ * why a pool couldn't be safely imported.
+ */
+typedef enum mmp_state {
+	MMP_STATE_ACTIVE = 0,		/* In active use		*/
+	MMP_STATE_NO_HOSTID		/* System hostid is not set	*/
+} mmp_state_t;
```

Comment:
I'll switch the names to ZPOOL_CONFIG_MMP_* and move then all in to ZPOOL_CONFIG_LOAD_INFO.